### ISSUE TITLE
One problem when i reproduce your result

### ISSUE THREAD

**Issue by wesley-stone** [charlesq34/frustum-pointnets#33]
First of all, it's really kind of you to share your code and result. I am trying to reproduce your result on kitti 3D detection dataset listed in the paper.

Currently, I have trained a model for 2D detection and gained similar mAP compared with yours in the paper. But when I use the pre-trained frustum model, both v1 and v2, I can only reach about 68% even in the easy part. I also find that in the precision-recall curve, the precision is only about 85% when recall is 0. I do not know if something is wrong. Can you please give some advice?

![image](https://user-images.githubusercontent.com/4548088/41659419-daa94ba6-74cb-11e8-9d1b-45bb0bcd59f4.png)

**Comment by ShiAGou** [comment]
I evaluated the pre-trained model using both ground truth 2D bounding box and rgb detection result. Indeed, the mAP of car with rgb detection result is close to those listed in the paper, but the strange thing is that when i use ground truth bounding box, mAP of 3D detection could not reach the expected value.
I suppose maybe Frustum model has also learned the bias of the 2D detector. Is this true?  should I use the results from 2D detector other than ground truth box during training? @charlesq34

**Comment by electricmanlpl** [comment]
Hi, @wesley-stone , I met the same issue, when I evaluate the F-Ponintnet with my 2D detection result, the AP curve is truncated in the precision axis.  I wonder how do you solve this problem?

**Comment by wesley-stone** [comment]
> Hi, @wesley-stone , I met the same issue, when I evaluate the F-Ponintnet with my 2D detection result, the AP curve is truncated in the precision axis.  I wonder how do you solve this problem?

**Comment by wesley-stone** [comment]
> Hi, @wesley-stone , I met the same issue, when I evaluate the F-Ponintnet with my 2D detection result, the AP curve is truncated in the precision axis.  I wonder how do you solve this problem?
I gave it up finally...
